test/integration/profile_security_question_test.go

```diff
@@ -0,0 +1,40 @@
+package integration
+
+import (
+    "context"
+    "fmt"
+    "testing"
+
```

Comment:
We'll need to import `github.com/dnaeon/go-vcr/recorder` here for the `warnSensitiveTest` helper below